backend/backend/views.py

```diff
@@ -54,9 +59,7 @@ class AuthenticatedBearerToken(DRFObtainAuthToken):
     permission_classes = [IsAuthenticated]
 
     def get(self, request, *args, **kwargs):
```

Comment:
Not directly linked with this PR, but I am not sure about the use of a `GET` request when the aim is to create a new resource